docs/classes/_bots_versionbot_.versionbot.html

```diff
@@ -380,7 +380,7 @@ <h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">ServiceLi
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/resin-io-modules/resin-procbots/blob/master/lib/bots/versionbot.ts#L754">bots/versionbot.ts:754</a></li>
+									<li>Defined in <a href="https://github.com/resin-io-modules/resin-procbots/blob/master/lib/bots/versionbot.ts#L803">bots/versionbot.ts:803</a></li>
```

Comment:
nit: should we suppress docs/* diffs? If so we could do it with a `.gitattributes` on the root dir:

```
.gitattributes
-------------

docs/* linguist-generated=true
```